components/Taxonomy/models.rst

```diff
@@ -4,116 +4,124 @@ Models
-| level           | How deep it is in the tree     | mixed                  |
-+-----------------+--------------------------------+------------------------+
-| createdAt       | Date when taxon was created    | \DateTime              |
-+-----------------+--------------------------------+------------------------+
-| updatedAt       | Date of last update            | \DateTime              |
-+-----------------+--------------------------------+------------------------+
-
-This model implements ``TaxonInterface``, it implements these extra methods:
-
-+------------------------------------+-------------------------------------+----------------+
-| Method                             | Description                         | Returned value |
-+====================================+=====================================+================+
-| hasChild()                         | Check whether the taxon has a child | boolean        |
-+------------------------------------+-------------------------------------+----------------+
-| addChild(TaxonInterface $taxon)    | Add child taxon                     | Void           |
-+------------------------------------+-------------------------------------+----------------+
-| removeChild(TaxonInterface $taxon) | Remove child taxon.                 | Void           |
-+------------------------------------+-------------------------------------+----------------+
-
-TaxonsAwareInterface
---------------------
-
-This interface should be implemented by models that support taxons.
-
-+------------------------------------+---------------------------------+--------------------+
-| Method                             | Description                     | Returned value     |
-+====================================+=================================+====================+
-| getTaxons($taxonomy = null)        | Get all taxons                  | TaxonInterface[]   |
-+------------------------------------+---------------------------------+--------------------+
-| setTaxons(Collection $collection)  | Set the taxons                  | Void               |
-+------------------------------------+---------------------------------+--------------------+
-| hasTaxon(TaxonInterface $taxon)    | Checks whether object has taxon | Boolean            |
-+------------------------------------+---------------------------------+--------------------+
-| addTaxon(TaxonInterface $taxon)    | Add a taxon                     | TaxonomyInterface  |
-+------------------------------------+---------------------------------+--------------------+
-| removeTaxon(TaxonInterface $taxon) | Remove a taxon                  | TaxonomyInterface  |
-+------------------------------------+---------------------------------+--------------------+
\ No newline at end of file
++-----------------+-----------------------------------------------------------------+
+| Attribute       | Description                                                     |
```

Comment:
We should keep this consistent. I see @tuka217 is using Property - which seems right.